TLM/TLM/Resources/whats_new.txt

```diff
@@ -1,10 +1,12 @@
-[Version] 11.7.2.0
-[Released] Nov 15th 2022
-[Link] tmpe-v11720-stable-15112022
+[Version] 11.7.3.0
+[Released] Dec 13th 2022
+[Link] tmpe-v11730-stable-13122022
 [Stable]
-[Meta] Compatibility patch for the game update 1.15.1-f4 (krzychu124)
-[New] Clear all TM:PE rules from selected node #692, #1668 (kianzarrin)
-[Fixed] Timed Traffic Lights wait = 0 issues #1497, #1674 (kianzarrin)
-[Updated] Minor changes to Timed Traffic Lights sensitivity slider #1670 (kianzarrin)
-[Updated] Pedestrian zone roads have now lower priority when setting up high priority road #1653 (kianzarrin)
+[Meta] Compatibility patch for the game update 1.16.0-f3 (krzychu124)
+[New] Improved bus/trolleybus stop path #1688, #1690 (kianzarrin)
```

Comment:
if this is visible in stable whats new panel then maybe rephrase it in a less abstract way:
buses pull to side faster at bus stops.

Reply:
maybe not faster but earlier?